metadata/com.hazelcast/hazelcast/5.2.1/jni-config.json

```diff
@@ -0,0 +1,166 @@
+[
+  {
+    "condition": {
+      "typeReachable": "com.hazelcast.internal.metrics.impl.MetricsService"
+    },
+    "name": "[Lcom.sun.management.internal.DiagnosticCommandArgumentInfo;"
+  },
+  {
+    "condition": {
+      "typeReachable": "com.hazelcast.internal.metrics.impl.MetricsService"
+    },
+    "name": "[Lcom.sun.management.internal.DiagnosticCommandInfo;"
+  },
+  {
+    "condition": {
+      "typeReachable": "com.hazelcast.internal.util.JVMUtil"
+    },
+    "name": "com.sun.management.VMOption$Origin",
```

Comment:
Those `com.sun.management`, `sun.management`, `java.management` etc causes trouble in the hibernate metadata. Could you please remove them from the metadata and try if Hazelcast works without them?

Reply:
- I submitted a new commit, `./gradlew clean test -Pcoordinates=com.hazelcast:hazelcast:5.2.1` is still normal.
- ![image](https://user-images.githubusercontent.com/20187731/203524482-6948499b-bcdb-4c13-8021-9b1849942bcc.png)